pkg/ebpf/debugfs.go

```diff
@@ -37,6 +47,13 @@ type KprobeStats struct {
 // event name format is p|r_<funcname>_<uid>_<pid>
 var eventRegexp = regexp.MustCompile(`^((?:p|r)_.+?)_([^_]*)_([^_]*)$`)
 
+func RefreshProbeStats() {
```

Comment:
I think this function is unnecessary, and you can put this function inline when you use it in the tracer.